src/bci_build/package/stunnel/README.md.j2

```diff
@@ -0,0 +1,71 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+
+## Description
+
+Stunnel is an open-source multi-platform application used to provide a universal
+TLS/SSL tunneling service.
+
+
+## How to use this Image
+
+The Stunnel Container Image launches `stunnel` by default using a minimal
+configuration file that only sets:
+- run in foreground
+- load further configuration files from `/etc/stunnel/conf.d`
+
+Custom configuration files should be placed into the directory
+`/etc/stunnel/conf.d`.
+
+The container entry point configures TLS/SSL automatically by setting the key
+and certificate to the values of the environment variables `STUNNEL_KEY` and
+`STUNNEL_CERT`. If one of the environment variables is unset, then the
+entrypoint defaults to `/etc/stunnel/stunnel.key` for `STUNNEL_KEY` and
+`/etc/stunnel/stunnel.pem` for `STUNNEL_CERT`.
+
+The entrypoint can setup a single service via environment variables so that the
+user doesn't have to write and mount their own configuration file. This can be
+setup via the environment variables `STUNNEL_SERVICE_NAME`, `STUNNEL_ACCEPT` and
+`STUNNEL_CONNECT`:
+
+- `STUNNEL_SERVICE_NAME`: name or otherwise unique identifier of the service
+  (used for documentation purpose only)
+
+- `STUNNEL_ACCEPT`: address on which new connections should be accepted, can be
+  either a hostname + port number or just a port number (then localhost is
+  assumed as the host)
+
+- `STUNNEL_CONNECT`: address on which the un-encrypted service is listening and
+  to which stunnel connects, can be either a hostname + port number or just a
+  port number (then localhost is assumed as the host)
+
+
+For example to create an SSL endpoint for a webserver listening on port `8000`
+on localhost, execute:
+
+```bash
+podman run --rm -d \
+    -p 8443:8443 \
+    -e STUNNEL_SERVICE_NAME=webserver \
+    -e STUNNEL_ACCEPT=0.0.0.0:8443 \
+    -e STUNNEL_CONNECT=0.0.0.0:8000 \
+    -v=path/to/server.pem:/etc/stunnel/stunnel.pem:Z \
+    -v=path/to/server.crt:/etc/stunnel/stunnel.crt:Z \
+    {{ image.pretty_reference }}
+```
+
+
+### Pitfalls
+
+The Stunnel Container Image is configured to launch `stunnel` as the `stunnel`
+user. Files mounted into a running container however belong to the `root` user
+by default. Set the file permissions of mounted files accordingly, so that
+non-owners and non-group members can read them.
```

Comment:
```suggestion
The Stunnel container image is configured to launch `stunnel` as the `stunnel`
user. But by default, files mounted into a running container belong to the `root` user. Set the file permissions of mounted files accordingly, so that
non-owners and non-group members can read them.
```